android/expoview/src/main/java/versioned/host/exp/exponent/ExperiencePackagePicker.kt

```diff
@@ -88,7 +87,6 @@ object ExperiencePackagePicker : ModulesProvider {
     NavigationBarPackage(),
     NotificationsPackage(),
     PermissionsPackage(),
-    SecureStorePackage(),
```

Comment:
Can you comment why `SecureStoreModule` isn't listed in the modules class section? Usually, we will look into that file to see which modules will be added to the experience. Without the comment, people may think that the `SecureStore` isn't added.